test/terraform/gke/main.tf

```diff
@@ -43,6 +43,6 @@ resource "null_resource" "kubectl" {
   provisioner "local-exec" {
     when       = destroy
     on_failure = continue
-    command    = "rm $HOME/.kube/${google_container_cluster.cluster[count.index].name}"
+    command    = "rm $HOME/.kube/consul-k8s*"
```

Comment:
has the location of the kubectl credentials moved? or is this a simplification so we don't have to use the variable?

Reply:
good question! This is because, in terraform 1.13+, they've removed an ability to reference resources from destroy-time provisioners. This means that you can't reference other resources (e.g. `google_container_cluster`) in these provisioners because terraform may not know what that reference resolves to if that resource has already been destroyed.

This change allows us to work around this new restriction. Since our GKE clusters and their respective kubeconfig files have the `consul-k8s` prefix, we can just delete those files based on that instead of referencing the name of the terraform resource.

Reply:
Ah thank you for the explanation!